Makefile

```diff
@@ -176,26 +176,20 @@ install_crds: _kustomize _init_devworkspace_crds
 	$(KUSTOMIZE) build config/crd | $(K8S_CLI) apply -f -
 
 ### install: Install controller in the configured Kubernetes cluster in ~/.kube/config
-install: _print_vars _kustomize _init_devworkspace_crds _create_namespace _eval_plugin_registry_url
-	mv config/cert-manager/kustomization.yaml config/cert-manager/kustomization.yaml.bak
-	mv config/service-ca/kustomization.yaml config/service-ca/kustomization.yaml.bak
-	mv config/base/config.properties config/base/config.properties.bak
-	mv config/base/manager_image_patch.yaml config/base/manager_image_patch.yaml.bak
-
-	envsubst < config/cert-manager/kustomization.yaml.bak > config/cert-manager/kustomization.yaml
-	envsubst < config/service-ca/kustomization.yaml.bak > config/service-ca/kustomization.yaml
-	envsubst < config/base/config.properties.bak > config/base/config.properties
-	envsubst < config/base/manager_image_patch.yaml.bak > config/base/manager_image_patch.yaml
+install: _print_vars _kustomize _init_devworkspace_crds _create_namespace _eval_plugin_registry_url generate_deployment
 ifeq ($(PLATFORM),kubernetes)
-	$(KUSTOMIZE) build config/cert-manager | $(K8S_CLI) apply -f - || true
+	$(K8S_CLI) apply -f config/current/kubernetes/combined.yaml || true
 else
-	$(KUSTOMIZE) build config/service-ca | $(K8S_CLI) apply -f - || true
+	$(K8S_CLI) apply -f config/current/openshift/combined.yaml || true
 endif
 
-	mv config/cert-manager/kustomization.yaml.bak config/cert-manager/kustomization.yaml
-	mv config/service-ca/kustomization.yaml.bak config/service-ca/kustomization.yaml
-	mv config/base/config.properties.bak config/base/config.properties
-	mv config/base/manager_image_patch.yaml.bak config/base/manager_image_patch.yaml
+### generate_deployment: Generate files used for deployment from kustomize templates, using environment variables
+generate_deployment:
+	config/generate-templates.sh
```

Comment:
Generate deployments invokes generate templates )
Maybe it makes sense to rename the script to generate_deployment as well?

Reply:
Yeah, I don't have a good term for "all of the yaml required to deploy the DevWorkspaceOperator" :)

Reply:
Is it necessary to have kustomize templates in config? It always confused me, so, if possible (which should be I think) I propose the following structure:
```
/deploy # at least cert-manager, eclipse-che use deploy folder to keep deployment (not k8s deployment) files
        # nginx ingress controller uses deployments https://github.com/nginxinc/kubernetes-ingress/tree/master/deployments
  /templates # it's where our kustomize templates live
  /deployment
    /openshift
    /kubernetes
  /current # it's where Makefile's static file live
```
WDYT?

Reply:
That would work, just gotta change some paths.

Reply:
It is done :tada: